lib/xandra.ex

```diff
@@ -214,13 +215,13 @@ defmodule Xandra do
   example, to start a pool of connections to Cassandra, the `:pool` option can
   be used:
 
-      Xandra.start_link(pool: DBConnection.Poolboy)
+      Xandra.start_link(pool: DBConnection.ConnectionPool)
 
   Note that this requires the `poolboy` dependency to be specified in your
```

Comment:
Should remove comment about requiring the poolboy dependency since DBConnection now uses its own pool

Reply:
removed